### PR TITLE
[EDU-910] - Update free tier limits

### PR DESCRIPTION
**Pull request by tbedford**
## Description

This PR raises the free-tier limits. The changes are based on the following:

* https://github.com/ably/website/blob/main/data/models/packages.yml
* Confluence page: https://ably.atlassian.net/wiki/spaces/DEVX/pages/2259616020/New+free+tier+limits

Jira tickets:

* Epic: [EDU-911](https://ably.atlassian.net/browse/EDU-911)
* [EDU-910](https://ably.atlassian.net/browse/EDU-910)

## Review

Check the limits page and the diff.

* [Limits page](https://ably-docs-edu-910-updat-44h24a.herokuapp.com/general/limits/)

### CODE REVIEW

File: content/general/limits.textile [hunk 1]
```diff
@@ -75,14 +75,14 @@ The quotas for an account can be considered a soft limit, with no restrictions c
...
-| "Bandwidth":#bandwidth (per hour in MiB) | 81.38 | 97.66 | Quota / 72 | 2.5x quota / 72 | Quota / 72 | 5x quota / 72 |\2. Custom |
-| "Bandwidth rate":#bandwidth (per second in KiB ) | - | 69 | - | Hourly * 2.5 | - | Hourly * 2.5 |\2. Custom |
+| "Peak connections":#peak-connections | 200 | 240 | Quota | 2.5x quota | Quota | 5x quota |\2. Custom |
+| "Peak channels":#peak-channels | 200 | 240 | Quota | 2.5x quota | Quota | 5x quota |\2. Custom |
+| "Total messages":#messages | 6,000,000 | 7,200,000 | Quota | 2.5x quota | Quota | 5x quota |\2. Custom |
+| "Messages":#messages (per hour) | 84,000 | 100,000 | Quota / 72 | 2.5x quota / 72 | Quota / 72 | 5x quota / 72 |\2. Custom |
+| "Message rate":#messages (per second) | - | 70 | - | Hourly * 2.5 | - | Hourly * 2.5 |\2. Custom |
+| "Total bandwidth":#bandwidth (GiB) | 11.5 | 13.8 | Quota | 2.5x quota | Quota | 5x quota |\2. Custom |
+| "Bandwidth":#bandwidth (per hour in MiB) | 160 | 190 | Quota / 72 | 2.5x quota / 72 | Quota / 72 | 5x quota / 72 |\2. Custom |
+| "Bandwidth rate":#bandwidth (per second in KiB ) | - | 110 | - | Hourly * 2.5 | - | Hourly * 2.5 |\2. Custom |
```
Comment (m-hulbert) [inline content/general/limits.textile:85]: These 3 rows are in GiB, MiB and KiB respectively, but the values that have been added are as GB, MB and KB.

Comment (m-hulbert) [inline content/general/limits.textile:85]: I think this should be 132KB or ~135KiB rather than 110?

File: content/general/limits.textile [hunk 2]
```diff
@@ -435,8 +435,8 @@ API request limits are the maximum number of REST API requests that can be made
 
 |_/2=. Limit |_\2=. Free |_\2=. Self-service |_\2=. Business (legacy) |_\2=. Enterprise |
 |_. Soft |_. Hard |_. Soft |_. Hard |_. Soft |_. Hard |_. Soft |_. Hard |
-| API requests (per hour) | 4,200 | 5,000 | 0.1 * hourly message soft limit | 0.1 * hourly message hard limit| 0.1 * hourly message soft limit | 0.1 * hourly message hard limit |\2. Custom |
-| API request rate (per second) | - | 20 | - | 50 minimum | - | 50 minimum |\2. Custom |
+| API requests (per hour) | 8,000 | 10,000 | 0.1 * hourly message soft limit | 0.1 * hourly message hard limit| 0.1 * hourly message soft limit | 0.1 * hourly message hard limit |\2. Custom |
+| API request rate (per second) | - | 27 | - | 50 minimum | - | 50 minimum |\2. Custom |
```
Comment (m-hulbert) [inline content/general/limits.textile:439]: This looks like it should remain 20 going by the website?

**Review by m-hulbert — CHANGES_REQUESTED** [review]
Just a couple of queries on a few values.

File: content/general/limits.textile [hunk 3]
```diff
@@ -401,9 +401,9 @@ Integration limits relate to the rate of "webhooks":/general/webhooks and the ra
 |_=. Limit |_=. Free |_=. Self-service |_=. Business (legacy) |_=. Enterprise |
 | "Webhook batch size":#webhook-batch | 50 | 100 | 100 | Custom |
 | "Webhook batch concurrency":#webhook-concurrency | 1 | 1 | 1 | Custom |
-| "Firehose message rate":#firehose-messages (per second) | - | - | - | Custom |
+| "External queue (Firehose) message rate":#firehose-messages (per second) | 50 | - | - | Custom |
```
Comment (m-hulbert) [inline content/general/limits.textile:404]: Small suggestion that we update the related heading name and paragraph for this limit if we're changing the name, but good to go without.

**Review by m-hulbert — APPROVED** [review]

File: content/general/limits.textile (hunk 3 above)
Comment (tbedford) [reply content/general/limits.textile:404]: Good idea. I'm going to slightly reword to I think.

**Pull request merged**